### PR TITLE
Fix embedded shopfront menu responsiveness

### LINKED ISSUE #2425: Tweak that might fix embedded shopfront on mobile - fix cart dropdown on small screen
## Description
I have been investigating embedded shopfront mobile and think it is possible that this issue you might be the only problem!

The cart drop down that enables access to edit card or checkout doesn't work on small screen so users get stuck. Once they are able to access edit cart or check out both screens seem fine on the small screen. Can we just fixed the responsiveness of this menu so that it works on small screens and see if that solves the problem?

## Expected Behavior
using [aus demo embedded shop](https://openfoodnetwork.org/user-guide/advanced-features/demo-embedded-shop/) as example 

On large or medium screens the cart dropdown shows buttons to edit cart, access checkout and the contents of cart e.g.

It is expected that the user would be able to access edit card and check out from this menu in small screen devices as well

## Actual Behavior

On small screen the cart drop-down only shows a thin black bar with no contents or buttons 

If you make the screen bigger to access the edit cart and checkout pages, but then smaller again to check them - I think they're actually fine (except maybe checkout button sliding off the bottom of edit cart) - not beautiful, but workable

## Steps to Reproduce

1. go to an embedded shop on mobile https://openfoodnetwork.org/user-guide/advanced-features/demo-embedded-shop/
2. put items in cart
3. click cart icon to try and access cart or checkout

## Context
Can't checkout on embedded shop on mobile

## Severity
I think s2 - preventing some users accessing checkout on mobile. Also, once this is fixed we will have a better idea of mobile path . .

## Your Environment
chrome, mac os and also android, chrome

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
#### What? Why?

Closes #2425

Improves the responsiveness of the navigation and fixes the cart dropdown in mobile view when using embedded shopfronts.

#### What should we test?

Embedded shopfronts UI on desktop and mobile.

#### Release notes

Improved mobile view for shops using embedded shopfronts. 

Changelog Category: Changed

### CODE REVIEW

File: app/assets/stylesheets/darkswarm/embedded_shopfront.css.scss [hunk 1]
```diff
@@ -1,27 +1,78 @@
 @import "typography";
 
+$large-menu-height: 4.6875rem;
+$medium-menu-height: 3rem;
+$gutter-width: 0.9375rem;
```
Comment (sauloperez) [inline app/assets/stylesheets/darkswarm/embedded_shopfront.css.scss]: :clap: variables should us bring consistency into the design in the long run.

**Comment by kirstenalarsen** [comment]
hi @Matt-Yorkley @sauloperez - I am happy to test this but am not sure of the testing process. I think I would need this on a staging server that has an embedded shop and not sure if aus does - @sstead ?

**Comment by mkllnk** [comment]
Staged on https://staging1.openfood.com.au/.

**Comment by sstead** [comment]
@kirstenalarsen to test an embedded shop on Staging1 you can go into Wordpress, edit this page (https://openfoodnetwork.org/user-guide/advanced-features/demo-embedded-shop/) and substitute the production URL for a staging1 URL temporarily.

**Comment by sstead** [comment]
@kirstenalarsen  I've embedded a staging1 shop into this page - https://openfoodnetwork.org/user-guide/advanced-features/demo-embedded-shop/

**Comment by kirstenalarsen** [comment]
thanks @sstead! I’ve been on a learning adventure and almost had one :) will complete my setup so that I have one for next time but will test that one!

**Comment by kirstenalarsen** [comment]
confirmarama!! this has done the trick and orders can now be placed in embedded shops on mobile!! yippee!! It's ugly but it works :) thanks @Matt-Yorkley

**Comment by RachL** [comment]
@kirstenalarsen did you happened to do some screenshots so we can move this to ready to go ?

**Comment by kirstenalarsen** [comment]
oh sorry - I didn't - and now it seems to have been changed back to the one coming from prod that doesn't work.

So i've dogey'd it and embedded it in openfoodnetwork.org as a post so it looks even worse, but screenshots can show that it works :)

image

image

**Comment by sauloperez** [comment]
so, is this ready? sorry, I'm not used to embedded shops

**Comment by kirstenalarsen** [comment]
yes it is ready - thanks @sauloperez

**Comment by sauloperez** [comment]
AWESOME!

**Comment by myriamboure** [comment]
@kirstenalarsen for your next tests here is our standardized testing template form :-) https://docs.google.com/document/d/16UZXJdemEI3EmcpFzJeuLchzkWKaaLoiJThr4cROig0/edit

**Pull request merged**